radar-upload-backend/src/main/java/org/radarbase/upload/resource/ProjectResource.kt

```diff
@@ -44,11 +45,13 @@ class ProjectResource(
 
     @GET
     @NeedsPermission(Permission.Entity.PROJECT, Permission.Operation.READ)
+    @Cache(maxAge = 300, isPrivate = true)
```

Comment:
This is a new feature from radar-jersey. Nice. Could you please tell me what does the `isPrivate `property in the annotation do?

Reply:
It is a client header, that tells the client (e.g., the browser) that it should not cache the same data for other users, but only for the current session: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control#cacheability